parser/src/parser.cpp

```diff
@@ -225,24 +250,62 @@ int main(int argc, char* argv[])
   if (vm.count("force"))
     cc::util::removeTables(db, SQL_DIR);
 
-  cc::util::createTables(db, SQL_DIR);
+  if (vm.count("force") || isNewDb)
+    cc::util::createTables(db, SQL_DIR);
 
   //--- Start parsers ---//
 
   cc::parser::SourceManager srcMgr(db);
   cc::parser::ParserContext ctx(db, srcMgr, compassRoot, vm);
   pHandler.createPlugins(ctx);
 
+  std::vector<std::string> reversedTopologicalOrder = pHandler.getTopologicalOrder();
```

Comment:
The original variable does not contain anything reversed. This can be confusing as there is a direct dependency for these two lines to always exist together.

Reply:
Well, it is only 2 lines of (simple) code, so I don't think it was that much confusing; naming the original variable `topologicalOrder` would have been much more confusing, since an *in-place* reversing is performed afterwards. Anyway, refactored it to use revere iterators in 4bfa0e3011222b14b1e78e95eee12c253c7edbf8.